apps/docs-snippets/test/fixtures/forc-projects/bytecode-input/Forc.toml

```diff
@@ -3,5 +3,8 @@ authors = ["Fuel Labs <contact@fuel.sh>"]
 license = "Apache-2.0"
 name = "bytecode-input"
 
-[dependencies]
-bytecode = { git = "https://github.com/FuelLabs/sway-libs", tag = "v0.19.0" }
+# TODO: Uncomment bytecode-related stuff
```

Comment:
@Dhaiwat10 I had to comment out everything related to `sway-libs` because it is not yet compatible with `forc@0.52+`. Consequently, the [code snippet](https://github.com/FuelLabs/fuels-ts/pull/2045/files#diff-1a11e2c177f1f12253aa42005f3290260aa32da5c9362433ee63dadd2aa5c5fa) is broken.

Can we remove this dependency to avoid the same happening against future updates? I don't think it's a one-time thing.

> cc @danielbate

Reply:
ping @Dhaiwat10

Reply:
The solution here might be to use a hardcoded snippet and create a follow-up issue to address the matter appropriately.

Reply:
Ok I did it.